lib/test.gi

```diff
@@ -536,7 +543,7 @@ end);
 ## 
 ##    testOptions := rec()   : Options to pass on to Test
 ##    earlyStop := false     : Stop once one test fails
-##    showProgress := true   : Show progress
+##    showProgress := "some" : Show some progress
```

Comment:
But this is an internal comment, summarizing the defaults -- if people need more clarity on what these options mean, they should look at the manual entry